openslides_backend/action/actions/poll/start.py

```diff
@@ -19,11 +21,43 @@ class PollStartAction(UpdateAction):
 
     def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         poll = self.datastore.get(
-            FullQualifiedId(self.model.collection, instance["id"]), ["state"]
+            FullQualifiedId(self.model.collection, instance["id"]),
+            ["state", "meeting_id"],
         )
         if poll.get("state") != Poll.STATE_CREATED:
             raise ActionException(
                 f"Cannot start poll {instance['id']}, because it is not in state created."
             )
         instance["state"] = Poll.STATE_STARTED
+
+        # restart projector countdown given by the meeting
+        if poll.get("meeting_id"):
+            meeting = self.datastore.get(
+                FullQualifiedId(Collection("meeting"), poll["meeting_id"]),
+                [
+                    "poll_couple_countdown",
+                    "poll_countdown_id",
+                ],
+            )
+            if meeting.get("poll_couple_countdown") and meeting.get(
+                "poll_countdown_id"
+            ):
+                countdown = self.datastore.get(
```

Comment:
After this line (same for the stop action) is "controlling a countdown". Maybe it can be put somewhere to the countdowns and be unified (like a function `control(countdown_id, action)`).

Reply:
Built a mixin ControlCountdown and use it in Start and Stop.